stdlib/src/binascii.rs

```diff
@@ -181,6 +181,74 @@ mod decl {
         }
         Ok((*c - 0x20) & 0x3f)
     }
+    #[derive(FromArgs)]
+    struct A2bqpArgues {
+        #[pyarg(any)]
+        data: ArgAsciiBuffer,
+        #[pyarg(named, default = "false")]
+        header: bool,
+    }
+    #[pyfunction]
+    fn a2b_qp(args: A2bqpArgues) -> PyResult<Vec<u8>> {
```

Comment:
`Argues` is a very peculiar way to write "args" or "arguments". Perhaps this struct should look like this:

```suggestion
    struct A2bQpArgs {
        #[pyarg(any)]
        data: ArgAsciiBuffer,
        #[pyarg(named, default = "false")]
        header: bool,
    }
    #[pyfunction]
    fn a2b_qp(args: A2bQpArgs) -> PyResult<Vec<u8>> {
```

@youknowone I think we should take a look at every `struct` that derives `FromArgs` so we can agree on a consistent naming scheme.

Reply:
I felt creating a new struct outside of function for everything which supports optional, named, etc is somewhat cumbersome to me. I think there's a better way for us to deal with this..?